sos/report/__init__.py

```diff
@@ -200,6 +201,12 @@ def add_parser_options(cls, parser):
                                 help="Escapes archived files older than date. "
                                      "This will also affect --all-logs. "
                                      "Format: YYYYMMDD[HHMMSS]")
+        report_grp.add_argument("--maxage", action="store",
+                                dest="maxage", default=None, type=int,
+                                help="Escapes archived files older (according "
+                                "to `mktime`) than this many hours. This will "
+                                "also affect --all-logs (sets the default for "
+                                "all plugins)")
```

Comment:
Would it be clear for users the option takes values in *hours*? Esp. if `--since` is primarily in days (default expected format is a day, optionally time)? Should not we rename it to, I dont know, `--maxhours`? (that sounds badly to me, I like the `--maxage` more).

I can easily be wrong here. I really dont know what users' perception would be here.

Reply:
I like the suggested change to `maxhours`, as it has the benefit of including the unit in the name and not force people to look at the description to figure it out.

I'll make the change.